### DISCUSSION TITLE
[Solved] Unable to set up rotary encoder

### DISCUSSION THREAD

**Discussion by Spacepadrille** [probonopd/MiniDexed#724] [Q&A] [answered]
First, this project is amazing and the sounds are really great ! I love this. I was successfull in setting the midi buttons and the navigation with midi buttons on my minilab 3. Great. But I also want to use a rotary encoder for the case I plug an other midi keyboard. But impossible to set up the thing. I soldered the wires coming from the rotary encoder on the GPIO 9, 10 and 11 (pins 19, 21 & 23) as mentioned in the wiki. I tried 3,3V and 5 V, no matters, no effect. I must miss something in the minidexed.ini file, but I don't know what... Some help ?

**Comment by diyelectromusic** [comment]
Is the encoder enabled in minidexed.ini? It usually is by default, but you might have disabled it if you were using MIDI buttons maybe?

More here: https://github.com/probonopd/MiniDexed/wiki/Files#ky-040-rotary-encoder

Kevin

**Comment by Spacepadrille** [comment, marked as answer]
Thanks a lot for you response. It was enabled. But now it is ok, it was just me confused with the pin report on the raspiaudio dac. Now all is ok. :-)
